### DISCUSSION TITLE
q-table records per page menu jumping around on first click

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#14660] [General - Components / Directives / etc]
Is there any way to prevent the q-menu in use in table pagination from jumping into position when it does not need to? I thought this was only happening in my code until I noticed the same behavior on the main q-table page: https://quasar.dev/vue-components/table#introduction

To reproduce:
1. go to any table example on the page like the first basic one (https://quasar.dev/vue-components/table#basic-usage)
2. click on the menu dropdown next to "Records per page"
3. watch as the menu jumps up and to the left slightly the first time used on the page, but not afterwards.

There is no discernable reason for this jump positioning to happen in this context, but it always does. If this is a bug, I will be happy to report it in issues, but if there is a way to control this in tables, I can add it there. Thanks.

**Comment by pdanpdan** [comment]
I think you should open an issue - please check if it happens the same with a normal menu (not the one for pagination in QTable)

**Reply by ssuess** [reply]
I checked already, it is not on the main q-menu page, only in table pagination (as far as I can tell). I will report it in issues.

**Reply by yusufkandemir** [reply]
#14661
